src/bob/pipeline/internals.clj

```diff
@@ -29,16 +29,36 @@
 
 ;; TODO: Reduce and optimize DB interactions to a single place
 
-(defn update-pid
+(defn commit-to-image
+  "Generates a new image from the diff of an existing container.
+  Returns the id of the new image"
+  [^String id ^List next-command ^List evars]
+  (let [repo (format "%s/%d" id (System/currentTimeMillis))
```

Comment:
Maybe we can skip the next-command and evars here. Not sure if they are needed for committing to image